src/frontend/apps/impress/src/components/BoxButton.tsx

```diff
@@ -1,4 +1,4 @@
-import { forwardRef } from 'react';
+import React, { forwardRef } from 'react';
```

Comment:
We pin to "react19.1.0", so adding "React" is not necessary, same for the others.

https://github.com/suitenumerique/docs/blob/bd79f84e070ebb42212859be1bbd7229c0ecbba2/src/frontend/package.json#L38

Reply:
Remove commit that make this change!